const/src/quotas.ts

```diff
@@ -69,6 +70,13 @@ export const quotaConfigs = {
     kind: 'workspace',
     category: 'count'
   },
+  document_count: {
+    name: 'Document Count',
+    description: 'Maximum number of documents in bytes that can be stored.',
+    default: 104_857_600, // 100 MB
+    kind: 'bot',
```

Comment:
should be a workspace limit
Also, should also be renamed `vector_db_storage` to be better aligned with the new pricing https://botpress-production.webflow.io/nextmktgpricing